internal/server/config/config.go

```diff
@@ -286,6 +286,8 @@ type Config struct {
 
 	RequestDurationDatastoreQueryCountBuckets []string
 	RequestDurationDispatchCountBuckets       []string
+
+	CheckTrackerEnabled bool
```

Comment:
Please update `DefaultConfig()` to include this explicitly as default=false and then add a unit test.

Reply:
Also, this tracker won't do anything if logging if disabled - please add validation that if tracker = true but logger = false, we error.